vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -134,15 +143,17 @@ $global:imagesToPull += @(
     "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.30.7-windows-hpc", # for k8s 1.30.x
     "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.31.1", # for k8s 1.31.x
     "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.31.1-windows-hpc", # for k8s 1.31.x
+
     # OMS-Agent (Azure monitor). Owner: ganga1980 (Ganga Mahesh Siddem)
     "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-3.1.24",
+
     # CNS (Container Networking Service) Owner: jaer-tsun (Jaeryn)
     "mcr.microsoft.com/containernetworking/azure-cns:v1.4.56",
-    "mcr.microsoft.com/containernetworking/azure-cns:v1.4.52",
```

Comment:
It seems like that v1.4.52 is still used in k8s < v1.28

Reply:
I was copying what the linux team had in their components.json. Will add it back.